cmd/kops/validate_cluster.go

```diff
@@ -117,70 +118,52 @@ func RunValidateCluster(f *util.Factory, cmd *cobra.Command, args []string, out
 		clientcmd.NewDefaultClientConfigLoadingRules(),
 		&clientcmd.ConfigOverrides{CurrentContext: contextName}).ClientConfig()
 	if err != nil {
-		return fmt.Errorf("Cannot load kubecfg settings for %q: %v\n", contextName, err)
+		return nil, fmt.Errorf("Cannot load kubecfg settings for %q: %v", contextName, err)
 	}
 
 	k8sClient, err := kubernetes.NewForConfig(config)
 	if err != nil {
-		return fmt.Errorf("Cannot build kube api client for %q: %v\n", contextName, err)
+		return nil, fmt.Errorf("Cannot build kubernetes api client for %q: %v", contextName, err)
 	}
 
-	// Do not use if we are running gossip
```

Comment:
Moved into validation

Reply:
what was the logic behind moving this? This seems to be an improvement of the code, not fixing the issue.

Wondering if we should put this in another PR. This needs to be tested on both Mac and Linux, since the OS behaves differently with Mac and caching dns.

Reply:
We're moving the code, not changing it AFAIK.

The reason for moving it is that we want this to be part of validation, I believe, including on a rolling-update.